projects/dotcms-field-elements/src/components/dot-date-time/dot-date-time.e2e.ts

```diff
@@ -0,0 +1,150 @@
+        await timeInput.press('2');
+        await page.waitForChanges();
+        expect(element.classList.contains('dot-valid')).toBe(true);
+        expect(element.classList.contains('dot-dirty')).toBe(true);
+        expect(element.classList.contains('dot-touched')).toBe(true);
+    });
+
+    it('it should not render hint', async () => {
+        element.setProperty('hint', '');
+        await page.waitForChanges();
+        const hint = await element.find('.dot-field__hint');
+        expect(hint).toBeNull();
+    });
+
+    it('it should have required as false', async () => {
+        element.setProperty('required', 'false');
+        await page.waitForChanges();
+        const required = await element.getProperty('required');
+        expect(required).toBeFalsy();
+    });
+
+    it('should show invalid range validation message, because date', async () => {
+        await dateInput.press('Tab');
+        await dateInput.press('Tab');
+        await dateInput.press('2');
+        await page.waitForChanges();
+        const errorMessage = await page.find('.dot-field__error-message');
+        expect(errorMessage.innerHTML).toBe('Value out of range');
+    });
+
+    it('should show invalid range validation message, because time', async () => {
+        element.setProperty('value', '2010-10-01 23:45:30');
+        await timeInput.press('0');
+        await page.waitForChanges();
+        const errorMessage = await page.find('.dot-field__error-message');
+        expect(errorMessage.innerHTML).toBe('Value out of range');
+    });
+
+    describe('emit events', () => {
+        it('should mark as touched when onblur', async () => {
+            await timeInput.triggerEvent('blur');
+            await page.waitForChanges();
+
+            expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                name: 'dt01',
+                status: {
+                    dotPristine: true,
+                    dotTouched: true,
+                    dotValid: true
+                }
+            });
+        });
+
+        it('should send status and value change and stop dot-input-calendar events', async () => {
+            const evt_statusChange = await page.spyOnEvent('_statusChange');
+            const evt_valueChange = await page.spyOnEvent('_valueChange');
+            await dateInput.press('2');
+            await timeInput.press('5');
+            await page.waitForChanges();
+            expect(spyStatusChangeEvent).toHaveReceivedEventDetail({
+                name: 'dt01',
+                status: {
+                    dotPristine: false,
+                    dotTouched: true,
+                    dotValid: true
+                }
+            });
+            expect(spyValueChange).toHaveReceivedEventDetail({
+                name: 'dt01',
+                value: '2019-02-20 05:10:00'
+            });
+            expect(evt_statusChange.length).toEqual(0);
+            expect(evt_valueChange.length).toEqual(0);
```

Comment:
please, add this test here:
https://github.com/dotCMS/core-web/blob/084f1acfd2625cafdb7dc9b2699d7ffd115abbc4/projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.tsx#L64

Reply:
done